code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/eventhub/EventHub.kt

```diff
@@ -129,6 +136,14 @@ internal class EventHub {
 
     init {
         registerExtension(EventHubPlaceholderExtension::class.java) {}
+        registerExtension(ConfigurationExtension::class.java) {
+            if (it == EventHubError.None) return@registerExtension
```

Comment:
nit: Let us also log successful registration of Configuration. It will be useful for debugging.

Reply:
The Extension base class [already logs successful case](https://github.com/adobe/aepsdk-core-android/blob/feature/eventhub/code/android-core-library/src/main/java/com/adobe/marketing/mobile/Extension.java#L75) so I did not log it here.